moto/awslambda/responses.py

```diff
@@ -359,3 +372,31 @@ def _put_code(self):
             return 200, {}, json.dumps(resp)
         else:
             return 404, {}, "{}"
+
+    def _get_function_concurrency(self, request):
+        function_name = self.path.rsplit("/", 2)[-2]
+        resp = self.lambda_backend.get_function_concurrency(function_name)
+
+        if resp:
```

Comment:
The if-else statements (for all three, get/put/delete) don't seem to be necessary - the backend always returns one value

Reply:
Ooops. I've forgot to validate function's name in the backend. We must return a 404 in this case, right ?

Reply:
Yes, correct. This is the error as returned from AWS:
```
botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the PutFunctionConcurrency operation: Function not found: [arn]
```